server/src/test/java/com/broadcom/lsp/cobol/usecases/TestReplacingWithDifferentPatternsShowsError.java

```diff
@@ -48,9 +48,153 @@
       "         NAME2.\n" + "           DISPLAY \"ABC\".\n" + "           MOVE 0 TO PARENT. ";
   private static final String NEW2_NAME = "NEW2";
 
-  private static final String MESSAGE =
-      "Syntax error on 'COPY' expected {<EOF>, ID, IDENTIFICATION, LINKAGE, LOCAL-STORAGE, WORKING-STORAGE, PROCEDURE, "
-          + "SCHEMA, END, FILE, MAP, SQL, 'EXEC SQL', EXEC, '01-49', '66', '77', '88'}";
+  private static final String MESSAGE = "Missing token IDMS at dataDescriptionEntryCopyIdms";
+
+  private static final String MESSAGE2 =
+          "Syntax error on 'REPLACING' expected {<EOF>, ASSOCIATED-DATA, ASSOCIATED-DATA-LENGTH, BOUNDS, CAPABLE," +
```

Comment:
Please, apply auto-formatting for this class to pass the style check

Reply:
In IDEA IDE you can enable `Operation sign on next line` option:
![image](https://user-images.githubusercontent.com/60617592/106878669-7febde00-66da-11eb-9474-24351df824fe.png)